android/app/src/main/java/com/textile/textilenode/TextileImageView.java

```diff
@@ -19,7 +21,7 @@
     private int forMinWidth;
     private ScaleType scaleType;
     private boolean needsRenderScaleType = false;
-    private boolean needsRenderImage = false;
+    private boolean needsRenderImage = true;
```

Comment:
such a hard time getting the white thumbnails solved. however, i noticed they are prone to display on the refresh after an app crash. it gave me the hunch that maybe a rerender wasn't being requested. i thought setting this to the default, needs rerender could be good. ideally, we could postpone line 75 to set it to false only after the bitmap has been successfully rendered (since it looks like 74 is async). this is a start though, default needs a rerender... not certain the implications of that though so think it through @asutula

Reply:
further testing and this at least solves white thumbnails for the instance where I can create them,

1. Open app
2. Do something that causes a JS error (in my case, i just did, https://github.com/textileio/photos/issues/1213#issuecomment-520177359)
3. Reload app.

With `needsRenderImage = false` thumbnails in the main view will 100% appear as white thumbs, but with `needsRenderImage = true` they will render fine.